runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -1230,7 +1512,9 @@ Java_java_lang_invoke_MethodHandleNatives_getMembers(
 	vmFuncs->internalEnterVMFromJNI(currentThread);
 	jint result = 0;
 	J9UTF8 *name = NULL;
-	J9UTF8 *sig = NULL;
+	char nameBuffer[256];
+	J9UTF8 *signature = NULL;
+	char signatureBuffer[256];
```

Comment:
Do you know what are the chances of this buffer being too small? given it is only used in this stack frame, it could be set to a bigger default (512) if there is a noticeable benefit in perf.

Reply:
I did some analysis for the Renaissance and DaCapo benchmark suites, neither the `nameBuffer` nor `signatureBuffer` were exceeded with 256 as the buffer size.